src/generators/operationGenerator.ts

```diff
@@ -435,9 +435,10 @@ function addClass(
     true /** shouldGuard */
   );
   const operationGroupClass = operationGroupFile.addClass({
-    name: className,
+    name: `${className}Impl`,
     docs: [`Class representing a ${className}.`],
-    isExported: true
+    isExported: true,
+    implements: [`${className}`]
```

Comment:
`className` should be renamed to `interfaceName`.

Reply:
In this place, it is easier for understanding to keep it a class name. Not changing here.